packages/strapi-admin/controllers/authentication.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Why changing the filename casing only for this controller?

Reply:
Just because it is a new one :) We can rename the other but it requires two commits (case sentiivity issue with git) and I didn't want to mess the PR readability :) I suggest we do this at the end of the feature to clean up the old file names that will still exist